lib/Auth/Process/LogUser.php

```diff
@@ -17,6 +17,12 @@
  */
 class LogUser extends \SimpleSAML\Auth\ProcessingFilter
 {
+
+    const AWS_ACCESS_KEY_ID_ENV = "DYNAMO_ACCESS_KEY_ID";
```

Comment:
Is there an example.env or such file where this is documented? Or maybe in the README?